Description: use older version of Nodejs module @mysticatea/eslint-plugin
 This patch partly reverts upstream git commit 41ff95e.
 .
 Also, bump minimal supported Nodejs release, to match code.
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2020-12-17
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/.eslintrc.yml
+++ b/.eslintrc.yml
@@ -1,15 +1,13 @@
 root: true
 
 extends:
-- plugin:@mysticatea/es2015
+  - plugin:@mysticatea/es2015
+  - plugin:@mysticatea/+modules
+  - plugin:@mysticatea/+node
 
-overrides:
-- files:
-  - src/**/*.js
-  - test/**/*.js
-  extends: plugin:@mysticatea/+modules
-  rules:
-    init-declarations: "off"
-    "@mysticatea/node/no-unsupported-features/es-syntax":
+rules:
+  init-declarations: "off"
+  "@mysticatea/node/no-unsupported-features/es-syntax":
     - error
-    - ignores: [modules]
+    - ignores:
+      - modules
--- a/package.json
+++ b/package.json
@@ -3,7 +3,7 @@
   "version": "2.1.0",
   "description": "Utilities for ESLint plugins.",
   "engines": {
-    "node": ">=6"
+    "node": ">=12"
   },
   "sideEffects": false,
   "main": "index",
